programmers/난이도별/level01.같은_숫자는_싫어/rockmiin.py

```diff
@@ -0,0 +1,16 @@
+def solution(arr):
+    answer=[]
+    answer.append(arr[0])
+    for i in range(1, len(arr)):
+        if(arr[i]!=arr[i-1]):
+            answer.append(arr[i])
+    # print(answer)
+    return answer
```

Comment:
```suggestion
    answer=[arr[0]]
    for v in arr:
        if(answer[-1] != v):
            answer.append(arr[i])
    return answer
```
이렇게 해도 될 것 같습니다~

혹은 [list.index()](https://wikidocs.net/14#index) 를 이용하는 방법도 있어요!

Reply:
감사합니다 :)